CONTRIBUTING.md

```diff
@@ -8,17 +8,28 @@ Please note we have a [code of conduct](https://github.com/reactdeveloperske/rea
 
 - Create a [new issue](https://github.com/reactdeveloperske/reactdevske-website/issues)
 - Comment on the issue (if you'd like to be assigned to it) - that way the issue can be assigned to you.
+  
+### Open issues
+- Go to [open issues](https://github.com/reactdeveloperske/reactdevske-website/issues)
+- select issue of your choice that is `open` or `need help` and is not assigned.
```

Comment:
Suggested edits:

Change to `Select an issue...`

Also they should check if the user has `up for grabs` label